components/cluster/pacemaker/pacemaker.p5m

```diff
@@ -96,23 +97,36 @@ file path=usr/lib/ocf/resource.d/pacemaker/o2cb
 file path=usr/lib/ocf/resource.d/pacemaker/ping
 file path=usr/lib/ocf/resource.d/pacemaker/pingd
 file path=usr/lib/ocf/resource.d/pacemaker/remote
-file path=usr/lib/pacemaker/attrd
-file path=usr/lib/pacemaker/cib
-file path=usr/lib/pacemaker/cibmon
-file path=usr/lib/pacemaker/crmd
-file path=usr/lib/pacemaker/lrmd
-file path=usr/lib/pacemaker/lrmd_internal_ctl
-file path=usr/lib/pacemaker/lrmd_test
-file path=usr/lib/pacemaker/pengine
-file path=usr/lib/pacemaker/stonith-test
-file path=usr/lib/pacemaker/stonithd
-file path=usr/lib/pkgconfig/pacemaker-cib.pc
-file path=usr/lib/pkgconfig/pacemaker-cluster.pc
-file path=usr/lib/pkgconfig/pacemaker-fencing.pc
-file path=usr/lib/pkgconfig/pacemaker-lrmd.pc
-file path=usr/lib/pkgconfig/pacemaker-pengine.pc
-file path=usr/lib/pkgconfig/pacemaker-service.pc
-file path=usr/lib/pkgconfig/pacemaker.pc
+
+file path=usr/lib/$(MACH64)/pacemaker/attrd
+file path=usr/lib/$(MACH64)/pacemaker/cib
+file path=usr/lib/$(MACH64)/pacemaker/cibmon
+file path=usr/lib/$(MACH64)/pacemaker/crmd
+file path=usr/lib/$(MACH64)/pacemaker/lrmd
+file path=usr/lib/$(MACH64)/pacemaker/lrmd_internal_ctl
+file path=usr/lib/$(MACH64)/pacemaker/lrmd_test
+file path=usr/lib/$(MACH64)/pacemaker/pengine
+file path=usr/lib/$(MACH64)/pacemaker/stonith-test
+file path=usr/lib/$(MACH64)/pacemaker/stonithd
+
+link path=usr/lib/pacemaker/attrd              target=/usr/lib/$(MACH64)/pacemaker/attrd
```

Comment:
Please, don't do this. Let's rather not ship 32-bit even though it will be ABI breakage. @pyhalov what do you think about such breakage in this case?

Reply:
Don't ship /usr/lib/pacemaker/* symlinks (especially these ones, which are incorrect - symlinks should be relative.

I'd shipped binaries in /usr/sbin or /usr/bin and libraries in /usr/lib/$(MACH64). But note, that pacemaker uses binaries from $(libdir)/pacemaker and as I understand searches for resources definitions in $(libdir)/ocf/resource.d . You should be careful with that, so that pacemaker could find resource descriptions of resource-agents package.
I'd prefer pacemaker-specific binaries to live where they are now (/usr/lib/pacemaker), it's consistent, for example, with sendmail. As for dlls, they should live in /usr/lib/$(MACH64) .

Was I clear? :)

Reply:
I hope I got it now. Just making clear the rules.